src/to_bytes.rs

```diff
@@ -126,12 +126,21 @@ pub unsafe fn guarded_transmute_to_bytes_many<T>(from: &[T]) -> &[u8] {
 /// }
 /// unsafe impl PodTransmutable for Gene {}
 ///
-/// assert_eq!(guarded_transmute_to_bytes_pod(&Gene {
+/// assert_eq!(safe_transmute_one_to_bytes(&Gene {
 ///                x1: 0x42,
 ///                x2: 0x69,
 ///            }),
 ///            &[0x42, 0x69]);
 /// ```
+pub fn safe_transmute_one_to_bytes<T>(from: &T) -> &[u8]
```

Comment:
Requirements in brackets, as opposed to a `where` clause

Occurs on:
  *	L135
  *	L188
  *	L297
  *	L308